src/sharing/components/ShareAutosuggest.jsx

```diff
@@ -28,6 +30,15 @@ const cozyUrlMatch = (input, contact) => {
   return urlInput.test(contact.url)
 }
 
+const groupNameMatch = (input, contactOrGroup) => {
+  if (contactOrGroup._type === DOCTYPE_GROUPS) {
+    const nameInput = new RegExp(input, 'i')
+    return nameInput.test(contactOrGroup.name)
+  }
+
+  return false
+}
```

Comment:
helpers ? utility ? Really needed in a jsx file?

Reply:
I also prefer to have the "break point" at the beginning of the method. Find this way more readable 
```
if (contactOrGroup._type !== DOCTYPE_GROUPS) return false
...
```

Reply:
I agree that this logic could go outside of the component even if it is used only here. It will be easier to test it

Reply:
about the `*Match`, I will leave them in here and we will move it and add tests when we will fix the fact to add a contact that has a cozy url